velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp

```diff
@@ -788,6 +788,18 @@ class ArrowBridgeArrayImportTest : public ArrowBridgeArrayExportTest {
     // reusable. So we don't check them in here.
     if constexpr (!std::is_same_v<T, std::string>) {
       EXPECT_FALSE(BaseVector::isVectorWritable(output));
+    } else {
+      size_t totalLength = 0;
+      bool isInline = true;
+      for (const auto& value : inputValues) {
+        if (value.has_value()) {
+          auto view = StringView(value.value());
+          totalLength += view.size();
+          isInline = isInline && view.isInline();
+        }
+      }
+      totalLength = isInline ? 0 : totalLength;
```

Comment:
This case https://github.com/facebookincubator/velox/blob/main/velox/vector/arrow/tests/ArrowBridgeArrayTest.cpp#L884-L897 is some of the strings are inlined and others are not.  And totalLength is 76. stringBuffers's capacity is 76 too.

Reply:
I see, it is a little bit wasteful because we can skip allocation for inlined even some strings are not inlined.  But it is like that before and can be improved later.

Reply:
Thanks.